scalafix-core/src/main/scala/scalafix/v1/SymbolInfo.scala

```diff
@@ -0,0 +1,21 @@
+package scalafix.v1
+
+import scala.meta.internal.{semanticdb => s}
+
+object SymbolInfo {
+  val empty = new SymbolInfo(s.SymbolInformation())
+}
+
+final class SymbolInfo private[scalafix] (
+    private[scalafix] val info: s.SymbolInformation
+) {
+  def isNone: Boolean = info.symbol.isEmpty
+  def sym: Symbol = Symbol(info.symbol)
+  def owner: Symbol = Symbol(info.symbol).owner
+  def name: String = info.name
+  def kind: SymbolKind = new SymbolKind(info)
+  def props: SymbolProperties = new SymbolProperties(info.properties)
```

Comment:
How about inlining `kind.isXXX` and `props.isXXX` into `SymbolInfo`, so that it's `info.isFinal` and `info.isMethod` instead of `info.props.isFinal` and `info.kind.isMethod`?

Reply:
https://github.com/scalameta/scalameta/pull/1750